README.md

```diff
@@ -79,6 +79,7 @@ A curated list of resources for learning about natural language processing, text
 - [Machine Learning for Text (2018)](https://www.springer.com/us/book/9783319735306)
 - [Towards Explainable NLP: A Generative Explanation Framework for Text Classification](https://arxiv.org/pdf/1811.00196.pdf)
 - [Natural Language Processing in Spanish](https://iaarbook.github.io/procesamiento-del-lenguaje-natural/)
+- [How to Write a Spelling Corrector - by Peter Norvig](https://norvig.com/spell-correct.html)
```

Comment:
Very interesting post, but I'm not sure it's a book. May be better under Blog Articles.

Reply:
Moved to Blog Articles, Papers, Case Studies